game/script-ch30.rpy

```diff
@@ -26,6 +26,14 @@ label ch30_visual_setup:
     #FALL THROUGH
 
 label ch30_init:
+
+    # Add to the total visits counter
+    $ persistent.jn_total_visit_count += 1
+
+    # Set the last visited date, if not already on record
+    if not persistent.jn_first_visited_date:
+        $ persistent.jn_first_visited_date = datetime.datetime.now()
```

Comment:
Might be better to do in `splashscreen` so we avoid checking for it on every cycle

Reply:
Can do, will get that added in next commit